app/javascript/packs/application.js

```diff
@@ -31,11 +34,39 @@ import "bootstrap";
 import { toggleRequired } from 'pluggins/checkbox';
 
 
```

Comment:
Lucien, the same goes for here: you should create a separate file to put all your functions, and import them in the application.js